package.json

```diff
@@ -23,9 +23,7 @@
   "dependencies": {
     "@theforeman/vendor": "^8.16.0",
     "graphql-tag": "^2.11.0",
-    "intl": "~1.2.5",
-    "jed": "^1.1.1",
-    "react-intl": "^2.8.0"
```

Comment:
[last time we tried to drop `react-intl`](https://github.com/theforeman/foreman/pull/8863), it exploded in our face:

- https://community.theforeman.org/t/foreman-develop-package-release-1166-failed/25839
- https://github.com/theforeman/foreman/pull/8882

And while this PR drops `webpack/assets/javascripts/react_app/common/I18n.js` (which I think was the offender last time), there are still references to `react-intl` in our code:

```console
% git grep react-intl            
webpack/assets/javascripts/react_app/common/i18nProviderWrapperFactory.js:import { IntlProvider } from 'react-intl';
webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/BuildModal.js:import { FormattedMessage } from 'react-intl';
webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js:import { FormattedMessage } from 'react-intl';
webpack/assets/javascripts/react_app/components/common/dates/IsoDate.js:import { FormattedDate, intlShape } from 'react-intl';
webpack/assets/javascripts/react_app/components/common/dates/LongDateTime.js:import { FormattedDate, intlShape } from 'react-intl';
webpack/assets/javascripts/react_app/components/common/dates/RelativeDateTime.js:import { FormattedRelative, intlShape } from 'react-intl';
webpack/assets/javascripts/react_app/components/common/dates/ShortDateTime.js:import { FormattedDate, intlShape } from 'react-intl';
```

Reply:
Those other "ordinary" occurences should be covered by ordinary [vendor-core dependency](https://github.com/theforeman/foreman-js/blob/86ed241e97cdae81c1d376ff5ddaedc38bc53b07/packages/vendor-core/package.json#L77) IMHO

Reply:
Well, let's try.

We know what to revert if stuff breaks ;)

Reply:
lol